Documentation/artifact-format-v3.md

```diff
@@ -487,7 +487,12 @@ tar, this is important.
 Compression
 ===========
 
-All file tree components ending in `.gz` in the tree displayed above should be
+Compression of the Artifact is optional.
```

Comment:
This says "optional", but then is not clear what suffix would apply if it is uncompressed. None at all?

Reply:
Hmm... Yeh, I get your point. I figured writing it it was implicit ofc, but if you are unsure whilst reading it it should probably be added.

Up to you @TheYoctoJester

Reply:
It should. Want me to do so?

Reply:
Done :) Have a look!